src/citrine/resources/workflow_executions.py

```diff
@@ -29,7 +29,7 @@ def __init__(self,
         self.session: Session = session
 
     def __str__(self):
-        return '<WorkflowExecution {!r}>'.format(self.uid)
+        return '<WorkflowExecution {!r}>'.format(str(self.uid))
```

Comment:
Couldn't you just remove the !r and get the same behavior?

Reply:
@andrewmillspaugh i think the `!r` adds the quotes, which we could also add back in, but this felt like the path of least resistance.

Reply:
Ah, I thought it was "render this thing using the __repr__ method"